study.md

```diff
@@ -75,7 +82,9 @@ Using Ruby's Hash methods, set a variable named `dale_keys` to `dale`'s keys.
 Additionally, set a variable named `dale_values` to `dale`'s values.'
 
 ```ruby
-# your answer here
+//Not really sure how to distinguish betweek keys and values
```

Comment:
We're just calling the Hash methods `keys` and `values` to get an array of the respective values back.